python/main.py

```diff
@@ -4,39 +4,104 @@
 from fastapi import FastAPI, Form, HTTPException
 from fastapi.responses import FileResponse
 from fastapi.middleware.cors import CORSMiddleware
+import db
+import hashlib
+from os.path import join, dirname, realpath
 
 app = FastAPI()
 logger = logging.getLogger("uvicorn")
-logger.level = logging.INFO
+logger.level = logging.DEBUG
```

Comment:
It's better we keep this as part of env settings. In production, we cannot have log level as DEBUG.